### ISSUE TITLE
Can't get presenter view to work in firefox

### ISSUE THREAD

**Issue by ashonit** [hakimel/reveal.js#1361]
When invoking presenter view in Firefox, the presenter window opens but it displays no content other than the timestamp stuck on 00:00:00.

This happens both on Linux (Kubuntu 14.04) and Windows 8.1, Firefox v40.0.3.

For convenience while testing I'm serving up my own presentation using Python's SimpleHTTPServer (rather than apache or similar) -- but it also happens on the example presentation at http://lab.hakim.se/reveal-js.

**Comment by dvenkatsagar** [comment]
You might want to check the speaker-notes section. It actually worked for me when I tried it from that.

**Comment by ashonit** [comment]
@dvenkatsagar: Not sure what you mean -- I was bringing up the speaker notes page but it wasn't populated with any content.

Anyway I think I worked it out. By default FF blocks popup windows, and selecting the manual "show https://..." doesn't seem to be enough -- you just get the almost blank window. When I went back and pressed S a second time on the presentation, the speaker view would be populated as expected. If I select "allow popups from the lab.hakim.se domain", the problem goes away (presenter view gets populated normally the first time it is brought up).
